app/controllers/sessions_controller.rb

```diff
@@ -17,7 +17,7 @@ def destroy
     id_token = session[:id_token]
     session.clear
     if id_token.present?
-      redirect_to "#{ENV.fetch("OKTA_ISSUER")}/v1/logout?id_token_hint=#{id_token}&post_logout_redirect_uri=#{request.base_url}"
+      redirect_to "#{ENV.fetch("OKTA_ISSUER")}/v1/logout?id_token_hint=#{id_token}&post_logout_redirect_uri=#{request.base_url}", allow_other_host: true
```

Comment:
What is `allow_other_host` enabling? and have we been doing this on other rails 7 upgrades and I'm just noticing it?

Reply:
It's a new option for Rails 7 that allows a redirect to some other website outside of the application. Those are now disallowed by default.